dbt/include/redshift/macros/utils/get_columns_spec_ddl.sql

```diff
@@ -0,0 +1,44 @@
+{% macro redshift__get_columns_spec_ddl() %}
+  {#- loop through user_provided_columns to create DDL with data types and constraints -#}
+  {%- if config.get('constraints_enabled', False) -%}
```

Comment:
Move this if statement to `adapters.sql` similar to this: https://github.com/dbt-labs/dbt-core/pull/6271/files#diff-7246f19f4d12042d172ac61b3d0c4564ea27dd8825870b35425ede1ea41625dbR28

Reply:
To ensure i'm following correctly -- this `if` can simply be removed, correct? I already have an if statement in `adapters.sql` that checks this config, and only calls `get_columns_spec_ddl()` if constraints are enabled, so this is redundant.

Reply:
made this change! Let me know if it looks off to you

Reply:
this looks good!